### ISSUE TITLE
Sugarcane: non-vanilla behaviour on fertilizers

### ISSUE THREAD

**Issue by ColinHDev** [pmmp/PocketMine-MP#4549]
## Description
While working on https://github.com/pmmp/PocketMine-MP/issues/4545, I stumbled upon a non-vanilla behaviour of pmmp's implementation of sugarcane reacting on fertilizers.
In this video, you can see the vanilla behaviour of sugarcane: https://youtu.be/WZJLEkgbNUM
By the following line (https://github.com/pmmp/PocketMine-MP/blob/master/src/block/Sugarcane.php#L85) you can tell, that fertilizers only work on the lower, but not on the middle part. 

## Possible Solution
The Bamboo class has the method `Bamboo::seekToTop() : Bamboo` (https://github.com/pmmp/PocketMine-MP/blob/master/src/block/Bamboo.php#L136#L143), which returns the highest Bamboo block of that single plant.
The Sugarcane class could implement a similar method `Sugarcane::seekToBottom() : Sugarcane`, which would return the lowest Sugarcane block of that plant. 
That way
```php
if(!$this->getSide(Facing::DOWN)->isSameType($this)){
    $this->grow();
}
```
(https://github.com/pmmp/PocketMine-MP/blob/master/src/block/Sugarcane.php#L85#L87) could be changed to
```php
$this->seekToBottom()->grow();
```

**Comment by dktapps** [comment]
Relates to #4931

**Comment by ShockedPlot7560** [comment]
Fixed in #4930
